apps/k8s-io/configmap-nginx.yaml

```diff
@@ -66,7 +66,7 @@ data:
       # ref: https://git.k8s.io/community/sig-architecture/api-review-process.md#voluntary
       server {
         server_name x-k8s.io;
-        listen 80 default_server;
```

Comment:
right, there can be only one default htat is k8s.io https://github.com/kubernetes/k8s.io/blob/c5c504d174a14f19807a8f4ebef0bcdf52c5544a/apps/k8s-io/configmap-nginx.yaml#L22-L23